examples/unwrap_script.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+""" This script will:
+- download 2-echo phase fieldmap data
+- do the subtraction
+- unwrap phase difference
+- save wrapped and unwrapped plot of first X,Y volume as myplot.png in current directory
+"""
+
+import os
+import numpy as np
+import matplotlib.pyplot as plt
+from zipfile import ZipFile
+import glob
+
+import nibabel as nib
+
+from shimmingtoolbox.unwrap import unwrap_phase
+
+
+def main():
+
+    # Download example data
+    url = 'https://github.com/shimming-toolbox/data-testing/archive/r20200709.zip'
+    filename = 'data-testing.zip'
+    # TODO: replace with download function
+    os.system('curl -o {} -L {}'.format(filename, url))
+    with ZipFile(filename, 'r') as zipObj:
+        # Extract all the contents of zip file in current directory
+        zipObj.extractall()
+    os.remove(filename)
+    # TODO: use systematic name for data-testing (could be in metadata of shimmingtoolbox
+    path_data = glob.glob('data-test*')[0]
+
+    # Open phase data
+    fname_phases = glob.glob(os.path.join(path_data, 'sub-fieldmap', 'fmap', '*phase*.nii.gz'))
+    nii_phase_e1 = nib.load(fname_phases[0])
+    nii_phase_e2 = nib.load(fname_phases[1])
+
+    # Subtract phase
+    phase_diff = nii_phase_e2.get_fdata() - nii_phase_e1.get_fdata()
+
+    # Open mag data
+    fname_phases = glob.glob(os.path.join(path_data, 'sub-fieldmap', 'fmap', '*magnitude*.nii.gz'))
+    nii_mag_e1 = nib.load(fname_phases[0])
+    nii_mag_e2 = nib.load(fname_phases[1])
+
+    # TODO: Convert to a function b0_map
+    # phasediff: (matlab code for 2 echoes)
+    # Z1(:,:,:) = mag_data(:,:,:,1).*exp(1i*ph_data(:,:,:,1));
+    # Z2(:,:,:) = mag_data(:,:,:,2).*exp(1i*ph_data(:,:,:,2));
+    # atan2(imag(Z1(:,:,:).*conj(Z2(:,:,:))),real(Z1(:,:,:).*conj(Z2(:,:,:))));
+    # Convert to radians (Assumes there are wraps)
+    phase_diff = - np.min(phase_diff) + phase_diff
+    phase_diff = phase_diff / 4096 * 2 * np.pi - np.pi
```

Comment:
There's the built-in and optimised numpy.interp funciton in numpy that does this directly:
```suggestion
    MAX_PHASE = 4096
    MIN_PHASE = 0
    phase_diff = np.interp(phase_diff, [MIN_PHASE, MAX_PHASE], [-np.pi, np.pi] )
```

Like this, it scales between -np.pi and np.pi, assigning MIN_PHASE to -np.pi and MAX_PHASE to np.pi. I'm not sure why you substract the minimum before hand so I'll let you rearrange the function with what you need.

Reply:
Not sure why but using `phase_diff = np.interp(phase_diff, [0, 4096], [-np.pi, np.pi])` does not work, as I already have it working another way i'll leave it be.

Reply:
Strange on my Linux and my Windows it works. I'll investigate.

Reply:
What does it give you?